README.md

```diff
@@ -5,4 +5,11 @@
 
 A place to store translation modules to convert custom wire formats to Honeycomb formatted data structures.
 
+IMPORTANT NOTE: most consumers of husky will probably want to use a backwards-compatible version
+of the opentelemetry protobufs. If this applies to you, add the following to your go.mod file:
+
+```go.mod
+replace go.opentelemetry.io/proto/otlp => github.com/honeycombio/opentelemetry-proto-go-compat/otlp v0.19.2
```

Comment:
New repo to hold the modified OTLP proto files is at github.com/honeycombio/opentelemetry-proto-go 👍🏻 
```suggestion
replace go.opentelemetry.io/proto/otlp => github.com/honeycombio/opentelemetry-proto-go/otlp v0.19.0
```